examples/bind-shopping.amp.html

```diff
@@ -0,0 +1,454 @@
+<!doctype html>
```

Comment:
are we going to make interpolation marker configurable? I know multiple CMS's that have problem with not being able to change it